src/open_inwoner/templates/pages/contactmoment/detail.html

```diff
@@ -1,21 +1,44 @@
 {% extends 'master.html' %}
-{% load i18n anchor_menu_tags card_tags file_tags grid_tags table_tags solo_tags form_tags button_tags %}
+{% load i18n grid_tags table_tags button_tags icon_tags %}
 
 {% block content %}
     {% if contactmoment %}
         {% render_grid %}
             {% render_column span=12 %}
-                {# Contactmoment/dashboard. #}
-                <h1 class="h1" id="title">{{ page_title }}</h1>
+                <h1 class="h1" id="title">{{ contactmoment.onderwerp }}</h1>
                 {% include "components/Dashboard/Dashboard.html" with metrics=metrics only %}
             {% endrender_column %}
-            {% render_column start=4 span=6 %}
+        {% endrender_grid %}
 
-                <div class="contactmoment__details">
-                {% contactmoment_table contactmoment zaak=zaak %}
+        {% render_grid %}
+            {% render_column start=4 span=6 %}
+                <div class="contactmoment">
```

Comment:
Here you should just add 1 wrapping Div (or Section) with class contactmoment - it is not necessary to add the same class over and over again to all the divs below if you already have it in a wrapper.